testapi.pm

```diff
@@ -835,21 +835,28 @@ Setting C<$no_regex> will cause it to do a plain string search.
 =cut
 
 sub wait_serial {
+    my $regexp = shift;
+    my %args   = OpenQA::Isotovideo::Utils::compat_args(
+        {
+            regexp           => $regexp,
+            timeout          => 90,
+            expect_not_found => 0,
+            quiet            => 0,
+            no_regex         => 0,
+            buffer_size      => undef,
+            record_output    => undef,
+            exclude_match    => undef
+        }, ['timeout', 'expect_not_found'], @_);
+    # hacky, but previous version converted 0 timeout to default
+    $args{timeout} ||= 90;
```

Comment:
I'm not sure, if we wanna keep that behavior. A quick `grep` didn't found such call with `timeout == 0`.

Reply:
Then I'm inclined not to keep it. But test writers should decide that.

Reply:
Isn't the default or whatever was assigned going to be assigned here? basically making this line not needed anymore

Reply:
From my point the wired thing with line is that timeout equal to `0` was mapped to default/90 as well and not only `undef`.

Reply:
Ok I removed that line now. So we have a none backward compatible change. But I didn't found such call in os-autoinst-distri-opensuse.